modules/serverless-blue-green-deploy.adoc

```diff
@@ -0,0 +1,101 @@
+[id="serverless-blue-green-deploy_{context}"]
+= Routing and managing traffic by using a blue-green deployment strategy
+
+You can safely reroute traffic from a live version of an app to a new version, by using a link:https://en.wikipedia.org/wiki/Blue-green_deployment[blue-green deployment strategy].
+
+.Procedure
+
+. Create and deploy an app as a Knative service.
+. Find the name of the first revision that was created when you deployed the service, by viewing the output from the following command:
++
+[source,terminal]
+----
+$ oc get ksvc <service_name> -o=jsonpath='{.status.latestCreatedRevisionName}'
+----
+. Add the following YAML to the service `spec` to send inbound traffic to the revision:
++
+[source,yaml]
+----
+...
+spec:
+  traffic:
+    - revisionName: <first_revision_name>
+      percent: 100 # All traffic goes to this revision
+...
+----
+. Verify that you can view your app at the URL output you get from running the following command:
++
+[source,terminal]
+----
+$ oc get ksvc <service_name>
+----
+. Deploy a second revision of your app by modifying at least one field in the `template` spec of the service and redeploying it. For example, you can modify the `image` of the service, or an `env` environment variable. You can redeploy the service by applying the service YAML file, or by using the `kn service update` command if you have installed the `kn` CLI.
+. Find the name of the second, latest revision that was created when you redeployed the service, by running the command:
++
+[source,terminal]
+----
+$ oc get ksvc <service_name> -o=jsonpath='{.status.latestCreatedRevisionName}'
+----
++
+At this point, both the first and second revisions of the service are deployed and running.
+. Update your existing service to create a new, test endpoint for the second revision, while still sending all other traffic to the first revision:
++
+.Example of updated service spec
+[source,yaml]
+----
+...
+spec:
+  traffic:
+    - revisionName: <first_revision_name>
+      percent: 100 # All traffic is still being routed to the first revision
+    - revisionName: <second_revision_name>
+      percent: 0 # 0% of traffic routed to the second revision
+      tag: v2 # A named route
+...
+----
++
+After you redeploy this service by reapplying the YAML resource, the second revision of the app is now staged. No traffic is routed to the second revision at the main URL, and Knative creates a new service named `v2` for testing the newly deployed revision.
+. Get the URL of the new service for the second revision, by running the command:
```

Comment:
re #1, updated

re #2, they're not really the same
> I guess the earlier step combined retrieving the URL and viewing the URL.

That's correct, but I don't think these are the same, since the first one is just the service in general, the second one is specific to a revision. Since SMEs and QE have already signed off on the technical content as is, I would prefer not to modify this unless there is a good reason to revisit later.